soliket/tests/test_mflike.py

```diff
@@ -8,8 +8,8 @@
 from distutils.version import LooseVersion
 
 import camb
-import mflike  # noqa
```

Comment:
Great! Now that we do not `import mflike` any more, I think we can also remove the dependence in `requirements.txt`.